src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub-ssh/ssh.tf

```diff
@@ -11,7 +11,14 @@ resource "kubernetes_config_map" "jupyterhub-ssh" {
   }
 
   data = {
-    "values.yaml" = "hubUrl: ${var.jupyterhub_api_url}"
+    "values.yaml" = <<-EOT
+      hubUrl: ${var.jupyterhub_api_url}
+      ssh:
+        config:
+          JupyterHubSSH:
+            debug: true
```

Comment:
As workaround for https://github.com/yuvipanda/jupyterhub-ssh/issues/80

Reply:
It seems strange to set debug to true, but I checked and that is the default value - https://github.com/yuvipanda/jupyterhub-ssh/blob/34cbd1813738299b08705231080d84dcaee56aad/helm-chart/jupyterhub-ssh/values.yaml#L31

Reply:
The `debug` isn't strictly necessary but including the `ssh.config` in the values.yaml is due to the issue I linked to above.